properties.yaml

```diff
@@ -392,3 +392,15 @@ IsolatedProjectsEnabled:
   documentation: |
     Enables support for Gradle's incubating 'isolated projects' feature
 
+StaticsReinitializeMode:
+  key: compose.reload.staticsReinitializeMode
+  type: enum
+  enumClass: org.jetbrains.compose.reload.core.StaticsReinitializeMode
+  target: [ application ]
+  default: "AllDirty"
```

Comment:
I would advise keeping the original behavior as default first and collecting user projects or code snippets which benefit from the changes, as well as projects and code snippets which would fail because of the 'AllDirty' mode.

On the one hand, having more up-to-date statics is desirable, on the other hand, re-initializing all statics (including final ones, including properties which technically are not dirty) could have severe unwanted side effects as well. I cannot predict the weight of both effects, therefore I suggest that 'real life applicability research' shall be the next step.